packages/@react-aria/calendar/src/useCalendarCell.ts

```diff
@@ -56,20 +57,42 @@ export function useCalendarCell(props: CalendarCellOptions, state: CalendarState
     }
   });
 
+  let {focusProps} = useFocus({
+    onFocus: (event) => {
```

Comment:
interesting, this kind of flips the way we've done this elsewhere
I don't know which I like better though, this vs potentially something like this?
```
let {focusProps} = useFocus()
onFocus: chain(focusProps.onFocus, (event) => ...
```
I'm not going to request it be changed here, just some food for thought

Reply:
I fixed this with: https://github.com/adobe-private/react-spectrum-v3/pull/188/commits/8361f607a029e1c291415b6e2795b76826a832be